jupyterbook_latex/nodes.py

```diff
@@ -50,19 +52,9 @@ def visit_H2Node(self, node):
     node.append(line_block)
 
 
-def depart_H2Node(self, node):
-    index = getIndex(self.body, self.h2Text)
-    if index:
-        self.body[index] = "\\Large " + self.h2Text
-    # else throw an error
-
-
-def visit_H3Node(self, node):
-    visit_H2Node(self, node)
-
-
-def depart_H3Node(self, node):
-    index = getIndex(self.body, self.h2Text)
+def depart_RootHeader(self, node):
+    index = get_index(self.body, node["header_text"])
+    size = "\\Large " if node["level"] <= 2 else "\\large "
```

Comment:
Thank you for transforming it to a single node, was planning this in a later release. I think we also need to accommodate sizes apart from these two, like H4, in case people have subsubsections, etc.

Reply:
@AakashGfude is this still a concern? Perhaps we can document the limitation for now. Can you coordinate this?

Reply:
@mmcky it has mimicked the previous implementation in this package. So, I think it is good for this PR. We should create an issue, and create a new PR for it.